fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -284,8 +284,7 @@ def initialize(user = nil, password = nil, use_shell_script = false, provider_sh
       # Xcode 6.x doesn't have the same iTMSTransporter Java setup as later Xcode versions, so
       # we can't default to using the better direct Java invocation strategy for those versions.
       use_shell_script ||= Helper.is_mac? && Helper.xcode_version.start_with?('6.')
-      use_shell_script ||= !ENV['FASTLANE_ITUNES_TRANSPORTER_USE_SHELL_SCRIPT'].nil?
-
+      FeatureManager.enable!(:use_iTMS_transporter_shell_script) if use_shell_script
```

Comment:
lets not enable things in the FeatureManager at this low of a code level and instead continue to switch on local variables.

we could add an `enabled_features` action to programmatically turn on/off features from the fastfile.